src/app/csv-collection/csv-collection-detail/csv-collection-detail.component.html

```diff
@@ -50,3 +66,92 @@ <h3 class="wipp-title">CSV Collection Detail</h3>
     </div>
   </div>
 </div>
+
+
+<div class="row" style="margin-top: 100px">
```

Comment:
There are a few `style` sections in this file that could be removed. Matching the `images-collection-detail` component style should prevent the need for custom style.